src/tests/test_dynamic_api/remote/test_views.py

```diff
@@ -575,7 +586,7 @@ def _raise_timeout(request):
         {
```

Comment:
wellicht een idee om even een comment in ieder blok te zetten waarin je beschrijft wat je precies test. Ik zou het net kunnen bedenken op basis van deze code.

Reply:
Done.